Mediapipe.Net/Calculators/FaceMeshCpuCalculator.cs

```diff
@@ -26,7 +26,7 @@ public class FaceMeshCpuCalculator : ICpuCalculator<List<NormalizedLandmarkList>
 
         public FaceMeshCpuCalculator()
         {
-            graph = new CalculatorGraph();
+            graph = new CalculatorGraph(System.IO.File.ReadAllText(graphPath));
```

Comment:
Try to see if you can use the [`ResourceManager`](https://github.com/vignetteapp/MediaPipe.NET/blob/master/Mediapipe.Net/Util/ResourceManager.cs)

Reply:
(actually do regular paths first and then we'll see)

Reply:
Since this is getting massive, we'll do that in a separate PR.

Reply:
Ok yeah why not, the title does describe what we've done so far.